docs/platform/one-click/how-to-deploy-grafana-with-one-click-apps/index.md

```diff
@@ -0,0 +1,67 @@
+| **Linode Plan** | Your Linode's [hardware resources](/docs/platform/how-to-choose-a-linode-plan/#hardware-resource-definitions). You can use any size Linode for your Grafana App as it uses minimal system resources. The minimum recommended memory is 255 MB and the minimum recommended CPU is 1. You can always [resize your Linode](/docs/platform/disk-images/resizing-a-linode/) to a different plan later if you feel you need to increase or decrease your system resources. *Required* |
+| **Linode Label** | The name for your Linode, which must be unique between all of the Linodes on your account. This name will be how you identify your server in the Cloud Manager’s Dashboard. *Required*. |
+| **Add Tags** | A tag to help organize and group your Linode resources. [Tags](/docs/quick-answers/linode-platform/tags-and-groups/) can be applied to Linodes, Block Storage Volumes, NodeBalancers, and Domains. |
+| **Root Password** | The primary administrative password for your Linode instance. This password must be provided when you log in to your Linode via SSH. It must be at least 6 characters long and contain characters from two of the following categories: lowercase and uppercase case letters, numbers, and punctuation characters. Your root password can be used to perform any action on your server, so make it long, complex, and unique. *Required* |
+
+When you've provided all required Linode Options, click on the **Create** button. **Your Grafana app will complete installation anywhere between 2-5 minutes after your Linode has finished provisioning**.
+
+## Getting Started after Deployment
+
+### Access Your Grafana Client
+
+Now that your Grafana One-Click App is deployed, you can log into Grafana to access all of its monitoring and visualization features.
+
+1. Open a browser and navigate to your [Linode's IP address](/docs/quick-answers/linode-platform/find-your-linodes-ip-address/) and port `3000`, for example, `http://192.0.2.0:3000/`. By default, Grafana will listen to port `3000`.
+
+    {{< note >}}
+The Grafana One-Click App will install and enable the [UFW firewall](/docs/security/firewalls/configure-firewall-with-ufw/) allowing TCP traffic on port `3000`.
+    {{</ note >}}
+
+1. Viewing Grafana's log in page, enter in `admin` as the *username* and the Grafana password you set in the [Grafana Options](#grafana-options) section of the Cloud Manager, as the *password*. Click on the **Log In** button.
+
+    ![Log into Grafana with your admin username and password.](grafana-login.png)
+
+1. Once you are logged into Grafana, complete your configurations by [adding a data source](https://grafana.com/docs/grafana/latest/features/datasources/add-a-data-source/#add-a-data-source), [creating dashboards](https://grafana.com/docs/grafana/latest/guides/getting_started/#create-a-dashboard), and [adding users](https://grafana.com/docs/grafana/latest/permissions/overview/).
+
```

Comment:
Looks like Grafana does install UFW, so we'll want to include that as a note or table here if you aren't opposed.